resources/function_help/json/if

```diff
@@ -4,10 +4,11 @@
   "description": "Tests a condition and returns a different result depending on the conditional check.",
   "arguments": [
 	{"arg":"condition","description":"the condition which should be checked"},
-	{"arg":"result_when_true","description":"the result which will be returned when the condition is True"},
-	{"arg":"result_when_false","description":"the result which will be returned when the condition is False"}
+	{"arg":"result_when_true","description":"the result which will be returned when the condition returns True or not null value"},
```

Comment:
@m-kuhn 
> In English they have very strict meanings which cannot be mixed and the two things need to be treated differently in the docs. I hope the French translation of the docs do also ;)

I know and don't worry this is taken into consideration when translating. It's easier for me to explain something concisely in French than English. I wasn't satisfied by my text (and even had a version with `null or zero value` before trying to factorize it :( ), reason why I asked for you rewording it better.

Ok for the text you propose. Given that it enumerates what is considered true or false, can't we just keep your three examples above. Isn't it enough? If not, which "all the other" examples are you talking about?